db/blob/blob_file_builder.cc

```diff
@@ -33,12 +34,13 @@ BlobFileBuilder::BlobFileBuilder(
     const std::string& column_family_name, Env::IOPriority io_priority,
     Env::WriteLifeTimeHint write_hint,
     std::vector<std::string>* blob_file_paths,
-    std::vector<BlobFileAddition>* blob_file_additions)
+    std::vector<BlobFileAddition>* blob_file_additions,
+    const std::shared_ptr<IOTracer>& io_tracer)
```

Comment:
Could you move the new parameter so it is between `write_hint` and `blob_file_paths` ? (That way, input parameters would precede output parameters, i.e. `blob_file_paths` and `blob_file_additions`.)